fedimint-wasm-tests/Cargo.toml

```diff
@@ -21,7 +21,7 @@ fedimint-mint-common = { path = "../modules/fedimint-mint-common" }
 fedimint-ln-client = { path = "../modules/fedimint-ln-client" }
 fedimint-wallet-client = { path = "../modules/fedimint-wallet-client" }
 js-sys = "0.3.61"
-wasm-bindgen = "=0.2.88" # must match the nix provided wasm-bindgen-cli version
+wasm-bindgen = "=0.2.89" # must match the nix provided wasm-bindgen-cli version
```

Comment:
Tests fail because the nix-provided version diverged as warned against in the comment.